src/proteus/dswx_hls.py

```diff
@@ -862,8 +886,20 @@ def create_landcover_mask(copernicus_landcover_file,
                                                size_y, size_x)
     del tree_binary_mask
 
-    tree_aggregate_sum = np.where(copernicus_landcover_array == 111,
-                                  tree_aggregate_sum, 0)
+    copernicus_forest = np.zeros_like(tree_aggregate_sum, dtype=np.uint8)
+    if copernicus_forest_classes is None:
+        copernicus_forest_classes = [20, 111, 113, 115, 116, 121, 123, 125,
+                                     126]
```

Comment:
Why don't we default this to the list of [20, 111, 113, 115, 116, 121, 123, 125, 126] in the run config instead?

Oh wait, it is actually set to this list as default (right?), so why this if needed?

Reply:
Yes, it is set as default in the runconfig. I also added it here just in case `copernicus_forest_classes` is set to `None` by the calling code.

Reply:
BTW, thank you for the fast review!